docs/notebooks/Calculating ground states on large scale systems with the quantum krylov subspace method.ipynb

```diff
@@ -0,0 +1,1037 @@
y+9tl3cLC4/j6goc8P13ylKlXzKB2RU6ZMYcKECaxdu5YzZ84QHx/P4sWL3dfPnTuXLVu2UFJSwqZNm7Ba9XWgV155hbvuuouIiAiKiopYv3497dq1A6CmpobPPvvsiupXVFSQm5vr3tYc9Wf4bTpGYy+uPSbZXlpOmw5RVBYWN/Mee8rOzsYSYlOSoWFtI+0+2R+HFu5xud3lYkfOKaztTilI5fn4+5u358DIDKr2gWqXlWPlA71eZ9WcvPnJ1/Rpo+YYwLqPf2vvAd7qGyWnqh0h9MSBZ+28Uvgk61OsCt6tNroHwOX3ASNrG2lnSSxVzq5A/Se6RoP3j1RwS3Wu9xv6mcwFjPN/p/tS44rwuLzKqbHj0GkiThQoSCV9wCguDb4sTfV6nU1z8sYn3zCgbanBqXQyFzDGV9WRWLR48DIXOF6hkZ61n3Cry/OGfhZsPcBbfaNsL+2Kw3k1DY/3cGjw4YlqPnWpe7etb9++RER4vkY1RtnCyu7duyksLGTu3LkAxMTEEBsby5AhQ9y/k5aWRlpaGjt37uSRRx5h7969WK1WnnnmGTZu3Mjo0aPJyclh7NixpKam0r17d15++WV+8pOf8Ic//OGyGXJzcxk0aJB724aF1d3udG/bi8sIa1/7YIZFR1Bd7Nt/7A0dOhQntcuxRmZoWNtICYs/JKrf9zwur6ms4G//WMSTO1coSOX5+Pubt+fAyAzK9gGrjZT/lGEN81xcq6i287Npk6k69rnxuaj/+Lf2HuCtvlEieqWS8Ic9WNt4vmA5K8sYMniQl1v5n9E9AC6/DxhZ20hXTfw1sXc/g83LPvD5J3sZ9NQPFaSSuYCR4p/aSfvUWz0ud1ZXsvqff+LZLX9VkEr6gJEG/rsEW0Q7j8vLK6v41c9mUPFVpoJUMhcwSptrEui/5DOsYZ7/oHfV2PnesCHgNP6kS8HWA7zVN0qXsfdy7b1p2MKjPK779nA2g26/yfBMF2VlZZGa6n3x1xtlCyuZmZkMHTrUvV1UVMSJEydITk72+N1x48YxZ84cDh48iKZpFBQUMHr0aACSkpJISkoiIyODa6+9lrfeeov//Oc/V7Sw0rdvX7KystzbmsPJwUl/dG+f+fQrUh79CZYQGxFdO1JTUYXL7tvBnZGR4bE6aVSGhrWN9ElFB14658Su1a8f3jaCzf9zP5HP/FxJroaPv795ew6MzKByH3j5XBnp5WE46qxQW9C4LkJj9evrlWSC+o9/a+8B3uobRdPgiVNWzjhdaHX2gVBcjOlSzot1erOhuQzuAXD5fcDI2kYqdYbwu5PhNPzkXygufjcmjhtuN8c+0Nr7gMp94EBlO1Z856JGq/9OZZs24Wz4/SzaP/n/lOSSPmCcjcWVfFAWRU2DuUC3yBBW/3u5snPuyVzAOItOuzhRo6HVOXoxBBfD2p9n9b4MJZmCrQd4q2+UCpeNR062xd5gTSfM4uTXN3XlRkXzQdDXCppC2cJKTEwMmzdvxul04nQ6uf/++0lISCAsLIzKykpOnTpFz576Ga/T09MpKioiLi4Ou91OQUEBOTk5JCUlcfz4cQ4cOEBiYiKbN29m4sSJV5whIiKi3iqUq8bBwTrX20vKyX3pLW7bvAhN09j35Bpf3X23lJQUrKG1T4ORGRrWNlKKBhWHYUM+2Kz65+jaWOGvQ60MjblBSSbwfPyH//mXxAxOwBYWQpcbevHez57zaT1vz4GRGVTuA/0c+snJcs7pJyjUgE5tLPxjeASxEVe+OuxrdR//1t4DvNU30vIyuP8jKLbrn60NscCgLlYW3tiVcFtXJZkaPv5gfB8wMoPK5x/gT6f1b4e7+E8quwv+K97Kz/r1UpZJ5gLGSQWqcuGlr/TXgWoXhFnhj4OtjOjm+UabUYyeC8Dl94HWOhdIdMJvMvWTVV48WXF0qIV/DG9DjyiZC1zUmucC/6jQT2R+tkrvAaEWGNDJyrNDuxAR0kVJpmCbC3irb6Tnz8DD+/SfNaDGBXf1tPGLxJ5YLIHzDVjKZlNTp05l48aNJCQk0KNHD/r06UPnzvoZjisrK5kxYwbnz58nJCSEyMhINm3aRMeOHQFYvXo1M2bMwGKx4HQ6efbZZ+nfvz+vvfYa77//Pi+//DIHDhxgxYoV/OIXv2hRzrwN75G34b0W399Az+BrFot+tvfpcfDGEVj5JfzjJkg22UmuP5rvv6/yDqQM/tA2RP8GqNxi+LIUurWFwV1Qcl6NSzHD+DNDBn/oHgVvjIXN38IfD8KiFPjhtapTeTLDGDRDBn+4uSu8fSvsLYQqJwyNgRjPTwgqZ4YxaIYM/vDLvjDlenjtCKz4Al64CQbKXMCUGfyhjQ3+Ngy+KoHcEn38D4kxzzdCXWSG8WeGDP5wdQS8NgZe+xb+cBCeHAi3Xac6lSczjEEzZPCHITGwa5x+0vLyGhgco/+7INAoW1gJDw/nzTffdG/PmjWLW265BYBOnTqRnp7e6G2nT5/O9OnTPS7//e9/z+9//3sARo0a1eJFFeF/3drCyG76woqXj1eKINC3g/6fCE42CwzQ18zp7vnxWhEE2obAD2JVpxAqdQmHEV31hZVwmQsEpd7t9f9EcLJaIPHCXKBntNosQo1wG4y+WnWKllH6dct1ZWZmMnjwYJ/9vffff99nf0sIIYQQQgghhBDCG9N8Afrhw4dVRxBCCCGEEEIIIYRoEtMsrASSqGtjuOPtP3Hu8BFs4WF88/qHHF61LegyCBHMzDAGzZBBiGCmegyqri9EsDPDGDRDBiGELKw0W9GBfHZNW4TFamXyB2l8uf4dHJXVQZdBiGBmhjFohgxCBDPVY1B1fSGCnRnGoBkyCBHsTHOOlUBlaxuGNSwEi03dQ2mGDEIEMzOMQTNkECKYqR6DqusLEezMMAbNkEGIYCVHrDRT5+Q4xm1eSKf+17N/ySZqyiqDMkOrY7UQ3rkdVUWlfi8V3rmd9+8WNihDo/XFFTPDGDRDhlbFwB4AjYxD6Q
```

Comment:
I would say more about what you are doing, how this circuit came about. That is connect it better with the rest of the notebook.